src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/AbstractOAuth2AuthenticationProvider.java

```diff
@@ -127,20 +127,36 @@ public OAuth2UserRecord getUserRecord(String code, @NotNull OAuth20Service servi
         throws IOException, OAuth2Exception, InterruptedException, ExecutionException {
         
         OAuth2AccessToken accessToken = service.getAccessToken(code);
-    
+//<<<<<<< HEAD
+
+        //logger.log(Level.FINE, "Autentication provider id: {0}", id);
+
+        final String userEndpoint = getUserEndpoint(accessToken);
+//=======
+
         // We need to check if scope is null first: GitHub is used without scope, so the responses scope is null.
         // Checking scopes via Stream to be independent from order.
         if ( ( accessToken.getScope() != null && ! getScope().stream().allMatch(accessToken.getScope()::contains) ) ||
              ( accessToken.getScope() == null && ! getSpacedScope().isEmpty() ) ) {
             // We did not get the permissions on the scope(s) we need. Abort and inform the user.
             throw new OAuth2Exception(200, BundleUtil.getStringFromBundle("auth.providers.insufficientScope", Arrays.asList(this.getTitle())), "");
         }
+//>>>>>>> develop
         
         OAuthRequest request = new OAuthRequest(Verb.GET, getUserEndpoint(accessToken));
         request.setCharset("UTF-8");
+//<<<<<<< HEAD
+
+        if (id.equals("microsoft")) {
+            request.addHeader("Accept", "application/json");
+        }
+
+        //final Response response = request.send();
+//=======
         service.signRequest(accessToken, request);
     
         Response response = service.execute(request);
+//>>>>>>> develop
```

Comment:
Should we remove all these `<<<<<<< HEAD` (and related) comments? These are from resolving merge conflicts?

Reply:
The branch has been update.